ffi/src/lib.rs

```diff
@@ -952,13 +954,13 @@ fn valkey_value_to_command_response(value: Value) -> RedisResult<CommandResponse
 /// * `route_bytes_len` must be 0 if `route_bytes` is null.
 /// * This function should only be called should with a `client_adapter_ptr` created by [`create_client`], before [`close_client`] was called with the pointer.
 #[no_mangle]
-pub unsafe extern "C" fn command(
+pub unsafe extern "C-unwind" fn command(
     client_adapter_ptr: *const c_void,
     channel: usize,
     command_type: RequestType,
-    arg_count: c_ulong,
-    args: *const usize,
-    args_len: *const c_ulong,
+    arg_count: usize,
+    args: *const *const u8,
+    args_len: *const usize,
     route_bytes: *const u8,
     route_bytes_len: usize,
```

Comment:
I ended up leaving the longer argument lists intact because not all languages support passing and returning structs/tuples around. See https://jakegoulding.com/rust-ffi-omnibus/tuples/. To support all languages, we would need to allocate more memory, and I don't think we want to do that.